src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -51,12 +51,27 @@
         Add a reference to a bug in the mask comment. May be specified multiple
         times to reference multiple bugs.
     """)
+mask_opts.add_argument(
+    '--email', dest='email', action='store_true',
+    help='spawn e-mail composer with prepared email for sending to mailing lists',
+    docs="""
+        Spawn user's preferred e-mail composer with a prepared e-mail for
+        sending a last rites message to Gentoo's mailing list (``gentoo-dev``
+        and ``gentoo-dev-announce``). The user should manually set the Reply-to
+        field for the message to be accepted by ``gentoo-dev-announce``.
+
+        For spawning the preferred e-mail composer, the ``xdg-email`` tool from
+        ``x11-misc/xdg-utils`` package.
+    """)
 
 
 @mask.bind_final_check
 def _mask_validate(parser, namespace):
     atoms = []
 
+    if namespace.email and not namespace.rites:
+        mask.error('you can send e-mail only when last riting packages')
```

Comment:
Generally try to avoid using `you` in error output and docs in general.

Something like `last rites required for email support` would be better.